app/src/androidTest/java/org/simple/clinic/drugs/PrescriptionRepositoryAndroidTest.kt

```diff
@@ -158,7 +162,7 @@ class PrescriptionRepositoryAndroidTest {
     // then
     val expectedPrescribedDrug = prescribedDrug.copy(
         durationInDays = 20,
-        updatedAt = prescribedDrug.updatedAt.plus(durationToAdvanceBy),
+        timestamps = timestamps.copy(prescribedDrug.updatedAt.plus(durationToAdvanceBy)),
```

Comment:
Can we use the parameter name `updatedAt` in the `copy` function. Currently we are updating `createdAt` because that's the first parameter in `Timestamps`.